### PR TITLE
modify DS logon is down language

### PR DESCRIPTION
**Pull request by phiden**
per @minafarzad modifying yellow alert box language for DS Logon being down

### CODE REVIEW

**Review by ncksllvn — APPROVED** [review]

File: src/platform/site-wide/user-nav/components/SignInModal.jsx [hunk 1]
```diff
@@ -50,7 +50,7 @@ class SignInModal extends React.Component {
               <div className="form-warning-banner">
                 <AlertBox
                   headline="DS Logon isn't working quite right"
-                  content="If you're having trouble signing in to Vets.gov using your DS Logon ID and password, please try again later. Or, you can try signing in with your MyHealth_e_Vet ID and password or through ID.me."
+                  content="If you're having trouble singing in to Vets.gov using your DS Logon username and password, please try again later. Or, you can try signing in with your My HealtheVet username and password or through ID.me."
```
Comment (U-DON) [inline src/platform/site-wide/user-nav/components/SignInModal.jsx]: Is this a typo? Why did it go from `signing` to `singing`?

File: src/platform/site-wide/user-nav/components/SignInModal.jsx [hunk 2]
```diff
@@ -50,7 +50,7 @@ class SignInModal extends React.Component {
...
                   isVisible
```
Comment (ncksllvn) [inline src/platform/site-wide/user-nav/components/SignInModal.jsx]: > If you're having trouble singing
I think that should still be `signing`, right?

Comment (phiden) [reply src/platform/site-wide/user-nav/components/SignInModal.jsx]: Ahhh because I copy-pasted. Fixing.

**Review by U-DON — APPROVED** [review]

**Review by edpaget — APPROVED** [review]
LGTM

**Pull request merged**